src/App.tsx

```diff
@@ -1,9 +1,91 @@
 import './App.scss';
+import React, { useEffect, useState } from 'react';
+import {
+  Route, Routes, NavLink, Navigate,
+} from 'react-router-dom';
+import 'bulma/css/bulma.css';
+import classNames from 'classnames';
+import { PeoplePage } from './elements/PeoplePage';
+import { Persone } from './react-app-env';
+import { NotFoundPage } from './elements/NotFoundPage';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+// eslint-disable-next-line max-len
+const API_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+const getPeople = async () => {
+  const res = await fetch(API_URL);
+
+  return res.json();
+};
+
+const App: React.FC = () => {
+  const [people, setPeople] = useState<Persone[]>([]);
+  const [activeLi, setActiveLi] = useState(false);
+
+  useEffect(() => {
+    getPeople().then(peple => {
+      setPeople(peple);
+    });
+  }, []);
```

Comment:
Fetch people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened, not when app is launched, move its fetching logic to `PeoplePage`